### PR TITLE
feat: add sender to instructions

### PR DESCRIPTION
**Pull request by stringhandler**
Adds sender to DAN instructions

NOTE: I also created new schema.rs and config file for generating the global db with diesel.  See dan_layer/storage_sqlite/README.md

### CODE REVIEW

File: dan_layer/core/src/models/instruction.rs [hunk 1]
```diff
@@ -78,6 +81,10 @@ impl Instruction {
         &self.args
     }
 
+    pub fn sender(&self) -> PublicKey {
```
Comment (sdbondi) [inline dan_layer/core/src/models/instruction.rs:84]: nit: this naming convention usually means a reference to the item is returned.
Suggest either `to_sender_public_key` or return a reference.

**Review by sdbondi — APPROVED** [review]
LGTM (utAck)

**Review by brianp — APPROVED** [review]
Reviewed and looks fine. ~I tried running the migration changes but diesel_cli is refusing to compile so going with ut for now.~

Just had to say it wouldn't work and then it worked. 👌🏻

File: dan_layer/core/src/models/instruction.rs (hunk 1 above)
Comment (stringhandler) [reply dan_layer/core/src/models/instruction.rs:84]: Ah yeah, I thought it was `Copy` for some reason....

**Review by SWvheerden — APPROVED** [review]

**Pull request merged**